optuna/visualization/_contour.py

```diff
@@ -286,12 +287,12 @@ def _generate_contour_subplot(
         y=y_indices,
         z=z,
         colorbar={"title": target_name},
-        colorscale=plotly.colors.PLOTLY_SCALES["Blues"],
+        colorscale=COLOR_SCALE,
         connectgaps=True,
         contours_coloring="heatmap",
         hoverinfo="none",
         line_smoothing=1.3,
-        reversescale=target is None and direction == StudyDirection.MAXIMIZE,
+        reversescale=target is not None or direction == StudyDirection.MINIMIZE,
```

Comment:
I'm wondering if we can make an utility for constructing `reversescale` (`f: (target, direction) -> bool`) in `visualization/_utils.py`. It can be used across backends although I'm not sure it's good to use an utility in `visualization/_utils.py` in matplotlib functions.

Reply:
Ah, good idea. I'll introduce your suggestion to this PR.